pfcpiface/up4.go

```diff
@@ -78,6 +78,17 @@ type meter struct {
 	downlinkCellID uint32
 }
 
+type tunnelPeer struct {
+	id uint8
+	// refCount list of F-SEIDs (UE sessions) using this tunnel peer
+	refCount set.Set
```

Comment:
This is really confusing. "Refcount" implies counter, comment says `list` and data type is `Set`.